requirements/pytorch/test.txt

```diff
@@ -16,3 +16,5 @@ psutil  # for `DeviceStatsMonitor`
 pandas  # needed in benchmarks
 fastapi
 uvicorn
+--extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
should we really hardcode the cpu version here?

Reply:
Probably not. I don't know how to set this based on the current setup.

Any ideas @Borda or @akihironitta?

Reply:
Not sure what you want to achieve... we shall not limit the dev environment.
We are referring to the CPU version in our CPU-based CI because it just wasting time for download and size or occupied cache to use the general torch version...

Reply:
The index url is different whether you are on a CPU or CUDA machine.

So writing `/cpu` in the requirements is not quite correct. How can we choose dynamically?